src/Client.js

```diff
@@ -326,8 +326,15 @@ Client.prototype._execute = function(method, path, data, query, options) {
         endTime
       )
       self._handleRequestResult(response, result, options)
-
-      return responseObject['resource']
+      
+      if (options.withFaunaMetrics) {
```

Comment:
```suggestion
      if (options.metrics) {
```

This is `fauna-shell`, so the `Fauna` is redundant. And it would be much more succinct to just call:

```
var client = new faunadb.Client({
  secret: "secret",
  metrics: true
})
```

Reply:
The `README` file should be updated to document this option.